modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -547,6 +548,44 @@ return view.extend({
 			so.value(index, '%s (Domain: %s, Local: %s)'.format(index, val.domain || '?', val.local || '?'));
 		});
 
+		o = s.taboption('srvhosts', form.SectionValue, '__srvhosts__', form.TableSection, 'srvhost', null,
+			_('Bind service records to a domain name: specify the location of services.') + (' See <a href="%s">RFC2782</a>.').format('https://datatracker.ietf.org/doc/html/rfc2782')
+			+ '<br />' + ('_service: _sip, _ldap, _imap, _stun, _xmpp-client, ... .') + (' (Note: while _http is possible, no browsers support SRV records.)')
```

Comment:
Sorry for the stylistic nitpick, it will be the last one I promise. Can you replace `...` with `…` (the single character `&hellip;` ellipsis) and change `asdf.com` and similar domains to `example.org` / `example.com` / `example.edu` (see https://en.wikipedia.org/wiki/Example.com)?